Base.rte/Devices/Special/Medikit/Medikit.lua

```diff
@@ -21,12 +21,14 @@ function Update(self)
 				end
 			end
 			if target and (target.Health < target.MaxHealth or target.WoundCount > 0) then
-				local strength = self.baseStrength + math.ceil(3000/(1 + math.abs(target.Mass - target.InventoryMass + target.Material.StructuralIntegrity) * 0.5));
+				local targetToughnessCoefficient = math.ceil((700/(1 + math.abs(target.Mass - target.InventoryMass + target.Material.StructuralIntegrity)))^1.6);
+				local strength = self.baseStrength + targetToughnessCoefficient;
+				
 				if target.Health < target.MaxHealth then
 					target.Health = math.min(target.Health + strength, target.MaxHealth);
 				end
 				if target.WoundCount > 0 then
-					target:RemoveWounds(math.ceil(strength * 0.15), true, false, false);
+					target:RemoveWounds(math.ceil(strength * 1), true, false, false);
```

Comment:
Is there a reason to multiply by `1` here?

Reply:
There isn't. 
I figure it gets optimized out.

Reply:
There's no guarantee this'll get optimised out. With LuaJIT enabled, it hopefully would, but it certainly wouldn't be optimised out when LuaJIT is disabled. It's not a big thing, but I'd remove the * 1.